openstack/cronus/values.yaml

```diff
@@ -187,3 +188,30 @@ ingress-nginx:
     - name: tls-secret-cronus
       secret:
         secretName: tls-secret-cronus
+
+simulator:
+  enabled: true
+  image:
+    name: cronus-exporter-repository
+    pullPolicy: IfNotPresent
+    tag: latest
+    imagePullSecrets: []
+  statefulSet:
+    enabled: true
+    podManagementPolicy: OrderedReady
+    annotations: {}
+    labels: {}
+    headless:
+      annotations: {}
+      labels: {}
+      servicePort: 80
+  updateStrategy: RollingUpdate
+  replicas: 1
+  nebula: 
```

Comment:
what should be under the nebula? Empty string?

Reply:
Yes, it gives the option to use a local nebula for debugging